recipes/jupyterlab-variableinspector/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "lckr_jupyterlab_variableinspector" %}
+{% set version = "3.0.6" %}
+
+package:
+  name: jupyterlab-variableinspector
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f80a742b7e56c613afd6759130963a9d574daecbd092014263e61303049e2839
+
+build:
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . --install-option="--skip-npm" -vv'
```

Comment:
@bollwyvl do you mind taking a quick look at this one before we merge?

Reply:
Thanks @ocefpaf  for the review

I used that option following a suggestion of @bollwyvl in that conda recipe : https://github.com/conda-forge/staged-recipes/pull/14182#discussion_r588899830 😉

Reply:
OK. I'm going to use that as a @bollwyvl seal of approval ;-p